### ISSUE TITLE
Ensure there are no redundant nodes passed to the visualization

### ISSUE THREAD

**Issue by biancadanforth** [electrolyfish/lightbeam-we#96]
When we capture and store first and third party requests, we should be careful to make sure we don't duplicate any entries that have already been stored.

**Comment by jonathanKingston** [comment]
Is this a duplicate of: https://github.com/electrolyfish/lightbeam-we/issues/89

Are you seeing duplicate websites?

**Comment by princiya** [comment]
Yes, # 89 is what I was wanting to be achieved. @biancadanforth do you agree?

**Comment by biancadanforth** [comment]
These are separate issues; #89 is to declutter the graph but does not explicitly address the presence, if any, of redundant nodes; this one makes sure there are no redundant nodes. I am not sure if there are any redundant nodes, but this issue was created to check this.

**Comment by jonathanKingston** [comment]
@biancadanforth worth checking... lets make this a less *redundant* p2 though.
